src/components/scenes/CryptoExchangeScene.js

```diff
@@ -323,7 +318,7 @@ export class CryptoExchangeScene extends Component<Props, LocalState> {
     const allowedWallets = []
     for (const id in wallets) {
       const wallet = wallets[id]
-      if (excludedCurrencyCode !== wallet.currencyCode) {
+      if (excludedCurrencyCode !== wallet.currencyCode && wallet.currencyCode !== 'ETH') {
         walletCurrencyCodes.push(wallet.currencyCode)
```

Comment:
@ahujsak Could you explain why ETH is being excluded here?

Reply:
ok, that leaked in from a fix from another bug. I am going to remove it, and it will be in the next PR.